src/System.Reflection.Extensions/pkg/System.Reflection.Extensions.pkgproj

```diff
@@ -5,8 +5,14 @@
   </PropertyGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
+    <ProjectReference Include="..\ref\4.0.0\System.Reflection.Extensions.depproj">
```

Comment:
@ericstj can correct me if I'm wrong but I don't believe we need this as it should automatically get pulled from the last stable package after the work he did.